hoomd/md/methods/methods.py

```diff
@@ -895,11 +906,12 @@ def __init__(self, filter, kT, alpha=None, tally_reservoir_energy=False):
 
         gamma = TypeParameter('gamma',
                               type_kind='particle_types',
-                              param_dict=TypeParameterDict(1., len_keys=1))
+                              param_dict=TypeParameterDict(default_gamma,
+                                                           len_keys=1))
```

Comment:
```suggestion
                              param_dict=TypeParameterDict(float,
                                                           len_keys=1))
        gamma.default = default_gamma
```
We shouldn't trust the parameters provided by the user.